cli/flags.rs

```diff
@@ -217,13 +217,13 @@ impl Flags {
       _ => {}
     }
 
-    match &self.unsafely_treat_insecure_origin_as_secure {
+    match &self.unsafely_ignore_certificate_errors {
       Some(ic_allowlist) if ic_allowlist.is_empty() => {
-        args.push("--unsafely-treat-insecure-origin-as-secure".to_string());
+        args.push("--unsafely-ignore_certificate_errors".to_string());
```

Comment:
`--unsafely-ignore-certificate-errors` ?

Reply:
Thanks, fixed in https://github.com/denoland/deno/pull/11628